StringCalculator/Delimiters/DelimitersHandler.cs

```diff
@@ -0,0 +1,59 @@
+using System.Collections.Generic;
+using System.Linq;
+using System.Text.RegularExpressions;
+
+namespace kata_string_calculator.Delimiters
+{
+    public class DelimitersHandler : IDelimiters
+    {
+        private const string OptionalArg = "//";
+        private const int CustomDelimiterIndex = 2;
+        private const char StartOfCustomDelimiter = '[';
+        private const string CustomDelimiterRegex = "(?<=\\[).+?(?=\\])";
+        
+        private string[] _delimiters = {",", "\n"};        
+        private string _numbersInStringInput = null!;
```

Comment:
```suggestion
        private string _numbersInStringInput = "";
```
Best to avoid setting things to `null` if you can manage it, especially if your code doesn't treat it differently to the default value (empty string in this case).